sql/sql_statistics.h

```diff
@@ -179,6 +182,7 @@ class Histogram
     case SINGLE_PREC_HB:
       return (uint) (((uint8 *) values)[i]);
     case DOUBLE_PREC_HB:
+    case JSON:
```

Comment:
So currently the code behaves identical for JSON type and DOUBLE precision. That's fine for a first iteration.